src/lang/ts/intermock.ts

```diff
@@ -321,6 +346,78 @@ function processArrayPropertyType(
   }
 }
 
+/**
+ * Process an array definition.
```

Comment:
This should be "Process a union property"

Reply:
Done